tests/unit/PolicyUtilsTest.ts

```diff
@@ -81,4 +91,56 @@ describe('PolicyUtils', () => {
             });
         });
     });
+    describe('getActivePolicies', () => {
+        beforeAll(() => {
+            Onyx.init({
+                keys: ONYXKEYS,
+                initialKeyStates: {
+                    [ONYXKEYS.SESSION]: {accountID: CARLOS_ACCOUNT_ID, email: CARLOS_EMAIL},
+                },
+            });
+        });
+
+        beforeEach(() => {
+            global.fetch = TestHelper.getGlobalFetchMock();
+            return Onyx.clear().then(waitForBatchedUpdates);
+        });
+        it('should return empty array', () => {
+            // Given a user with a single archived paid policy.
+            const policies = {
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                1: {
+                    ...createRandomPolicy(1, CONST.POLICY.TYPE.CORPORATE),
+                    role: '',
+                },
+            };
+            const result = PolicyUtils.getActivePolicies(policies as OnyxCollection<Policy>, true);
+            // The result should be empty array since the policies contains only archived paid policy.
+            expect(result.length).toBe(0);
+        });
+        it('should return array contains policy which has id = 1', () => {
+            // Given a user with only a paid policy.
+            const randomPolicy1 = {...createRandomPolicy(1, CONST.POLICY.TYPE.CORPORATE), pendingAction: null};
+            const policies = {
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                1: randomPolicy1,
+            };
+            const result = PolicyUtils.getActivePolicies(policies as OnyxCollection<Policy>, true);
+            // The result should contain the mock paid policy.
+            expect(result).toContainEqual(randomPolicy1);
+        });
+        it('should return empty array', () => {
+            // Given a user with only one control workspace which is pending delete.
+            const policies = {
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                1: {
+                    ...createRandomPolicy(1, CONST.POLICY.TYPE.CORPORATE),
+                    pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
+                },
+            };
+            const result = PolicyUtils.getActivePolicies(policies as OnyxCollection<Policy>, true);
+            // The result should be empty array since there is only one policy which has pendingAction is 'delete'.
```

Comment:
```suggestion
            // The result should be an empty array since there is only one policy which is pending deletion, so we have no active paid policies.
```